elements/lisk-transactions/src/8_transfer_transaction.ts

```diff
@@ -151,7 +151,7 @@ export class TransferTransaction extends BaseTransaction {
 				new TransactionError(
 					'Amount must be a valid number in string format.',
 					this.id,
-					'.amount',
+					'asset.amount',
```

Comment:
```suggestion
					'.asset.amount',
```

Reply:
Notice and fixed before getting your comments :)